docs/values.md

```diff
@@ -281,6 +281,8 @@ authtoken = "<authtoken>"
 print(base64.b64encode(authtoken.encode("ascii")))
 ```
 
+**Note:** From release v1.12.1 onwards, AKO supports reading Avi controller credentials from existing `avi-secret` in `avi-system` namespace. If username and either of password or authtoken are not specified, avi-secret will not be created as part of Helm installation. AKO will assume that avi-secret already exists in avi-system namespace and will reference it.
```

Comment:
We should say `ako installed namespace`. Customer might install AKO in different namespace.

Reply:
Did we test HA deployment?

Reply:
Do we know it can be used as part of AKO operator installed AKO?

Reply:
@akshayhavile I have updated it to `ako installed namespace`. HA deployment should not be an issue since it is just one more replica which will use the same secret. Also, for AKO Operator, we are required to create avi-secret before applying akoconfig object to create ako deployment. Currently, we only support avi-system namespace with ako-operator.